core/include/webview/detail/backends/cocoa_webkit.hh

```diff
@@ -84,39 +84,12 @@ private:
 class cocoa_wkwebview_engine : public engine_base {
 public:
   cocoa_wkwebview_engine(bool debug, void *window)
-      : m_debug{debug},
+      : m_app{get_shared_application()},
         m_window{static_cast<id>(window)},
         m_owns_window{!window} {
-    auto app = get_shared_application();
-    // See comments related to application lifecycle in create_app_delegate().
-    if (!m_owns_window) {
-      set_up_window();
-    } else {
-      // Only set the app delegate if it hasn't already been set.
-      auto delegate = objc::msg_send<id>(app, "delegate"_sel);
-      if (delegate) {
-        set_up_window();
-      } else {
-        m_app_delegate = create_app_delegate();
-        objc_setAssociatedObject(m_app_delegate, "webview", (id)this,
-                                 OBJC_ASSOCIATION_ASSIGN);
-        objc::msg_send<void>(app, "setDelegate:"_sel, m_app_delegate);
-
-        // Start the main run loop so that the app delegate gets the
-        // NSApplicationDidFinishLaunchingNotification notification after the run
-        // loop has started in order to perform further initialization.
-        // We need to return from this constructor so this run loop is only
-        // temporary.
-        // Skip the main loop if this isn't the first instance of this class
-        // because the launch event is only sent once. Instead, proceed to
-        // create a window.
-        if (get_and_set_is_first_instance()) {
-          objc::msg_send<void>(app, "run"_sel);
-        } else {
-          set_up_window();
-        }
-      }
-    }
```

Comment:
This whole block has been moved into standardly named methods so that all platform class constructors become:
```
 m_window_init();
 m_window_settings(debug);
 dispatch_size_default(m_owns_window);
```

Reply:
The change seems reasonable to me.

I would however remove the `m_` prefix in the method names.

Reply:
Removed `_m` prefix in latest commit.